src/components/DefineCompany/BusinessContactInfo.vue

```diff
@@ -0,0 +1,154 @@
+<template>
+  <div>
+    <div v-if="!isEditing">
+      <div>Email Address: {{contact.email}}</div>
+      <div>Phone: {{contact.phone}}   ext:{{contact.phoneExtension}}</div>
+    </div>
+    <v-card flat class="business-contact-container">
+      <v-form v-model="formValid"  v-if="isEditing" ref="form">
+        <v-row>
+          <v-col cols="12">
+            <v-text-field
+              filled
+              label="Email Address"
+              req
+              persistent-hint
+              :rules="emailRules"
+              v-model="contact.email">
+            </v-text-field>
+          </v-col>
+        </v-row>
+        <v-row>
+          <v-col cols="12">
+            <v-text-field
+              filled
+              label="Confirm Email Address"
+              req
+              persistent-hint
+              :error-messages="emailMustMatch()"
+              v-model="contact.confirmEmail">
+            </v-text-field>
+          </v-col>
+        </v-row>
+        <v-row>
+          <v-col cols="12" md="6">
+            <v-text-field
+              filled
+              label="Phone Number"
+              persistent-hint
+              hint="Example: (555) 555-5555"
+              type="tel"
+              v-mask="['(###) ###-####']"
+              v-model="contact.phone"
+              :rules="phoneRules">
+            </v-text-field>
+          </v-col>
+          <v-col cols="3">
+            <v-text-field
+              filled label="Extension"
+              persistent-hint
+              :rules="extensionRules"
+              v-mask="'#####'"
+              v-model="contact.phoneExtension">
+            </v-text-field>
+          </v-col>
+        </v-row>
+      </v-form>
+      </v-card>
+  </div>
+</template>
+
+<script lang="ts">
+// Libraries
+import { Component, Vue, Prop, Watch, Emit } from 'vue-property-decorator'
+import { mask } from 'vue-the-mask'
+
+// Interfaces
+import { BusinessContactIF } from '@/interfaces'
+
+@Component({
+  directives: { mask }
+})
+export default class BusinessContactInfo extends Vue {
+  // Props
+ @Prop({
+   default: ():BusinessContactIF => ({
+     email: '',
+     confirmEmail: '',
+     phone: '',
+     phoneExtension: ''
+   })
+ })
+  private initialValue: BusinessContactIF
```

Comment:
Just curious, do we need to re-initialize these values. From what i can see they are defined in the store as strings ( not null or undefined ) and we then pass down that from DefineCompany into the component. 

Was it throwing errors when trying to just pass the prop as `@Prop() private initialValue: BusinessContactIF` ? If there was a type error, we could also tell typescript we can be sure it isn't null or undefined by added a `!` at the end of the property name.

Reply:
I will check that

Reply:
I tested that. default value is not needed. Removed that from the prop

Reply:
Right on, thanks!